packages/desktop-client/src/components/accounts/Account.js

```diff
@@ -567,13 +567,13 @@ function SelectedTransactionsButton({
             }
 
             if (scheduleId) {
-              history.push(`/schedule/edit/${scheduleId}`, {
+              history.push(`/schedules/edit/${scheduleId}`, {
                 locationPtr: history.location
               });
             }
             break;
           case 'link-schedule':
-            history.push(`/schedule/link`, {
+            history.push(`/schedules/link`, {
```

Comment:
I'd avoiding changing URLs unless you have a strong reason to. This makes it inconsistent with others and the singular makes more sense because you are linking an individual schedule.

Reply:
Good point! I didn’t see that other things used the singular so it makes sense to stick with the previous scheme here. I made this change because I noticed that the “schedules” button in the sidebar was getting un-highlighted when clicking on the button to edit the schedule. I think that may be a deeper issue though where the link is styling itself based on the URL of the frontmost modal, not the actual page that’s open.

Reply:
(maybe this is a reason to change up how that system works though, I’ve noticed that reloading while a modal is open crashes the app until you change the URL)

Reply:
Oh I see, good point. Yeah there's some weirdness there! Overall if you land some stuff separately like Matiss suggested it will be easier to review :)